### PR TITLE
Fix debug script

### PR DESCRIPTION
**Pull request by Gusted**

### CODE REVIEW

File: src/inject/utils/log.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-declare const __DEBUG__: boolean;
-const DEBUG = __DEBUG__;
+declare const __WATCH__: boolean;
```
Comment (alexanderby) [inline src/inject/utils/log.ts:1]: DEBUG should be still there. DEBUG is used for logging info while you debug, WATCH is used for watching for code changes.

File: tasks/build.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ async function release() {
 async function debug({watch}) {
     log.ok('DEBUG');
     try {
-        await runTasks(standardTask, {production: false});
+        await runTasks(standardTask, {production: watch});
```
Comment (alexanderby) [inline tasks/build.js:38]: It should be `{debug: true, watch}`

File: tasks/bundle-api.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ async function bundleAPI({production}) {
                 cacheRoot: production ? null : `${fs.realpathSync(os.tmpdir())}/darkreader_api_typescript_cache`,
             }),
             rollupPluginReplace({
-                '__DEBUG__': 'false',
+                '__WATCH__': 'false',
```
Comment (alexanderby) [inline tasks/bundle-api.js:34]: `__DEBUG__` should be in this case

File: tasks/bundle-js.js [hunk 4]
```diff
@@ -108,8 +108,8 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
                 cacheRoot: production ? null : `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache`,
```
Comment (alexanderby) [inline tasks/bundle-js.js:108]: So, there should be no more `production` property, but `debug` with inverted logic (if debug, use cache etc).

File: tasks/bundle-js.js [hunk 5]
```diff
@@ -108,8 +108,8 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
...
             }),
             rollupPluginReplace({
-                '__DEBUG__': production ? 'false' : 'true',
-                '__PORT__': production ? '-1' : String(PORT),
+                '__WATCH__': production ? 'true' : 'false',
```
Comment (alexanderby) [inline tasks/bundle-js.js:111]: Both `__DEBUG__` and `__WATCH__` should be there. Search for `production` all over the code and replace it with `debug` and add `watch` where necessary.

File: src/inject/utils/log.ts (hunk 1 above)
Comment (Gusted) [reply src/inject/utils/log.ts:1]: .... This getitng me headache

File: tasks/bundle-css.js [hunk 6]
```diff
@@ -41,7 +41,7 @@ module.exports = createTask(
 ).addWatcher(
     ['src/**/*.less'],
     async () => {
-        await bundleCSS({production: false});
+        await bundleCSS({debug: false});
```
Comment (alexanderby) [inline tasks/bundle-css.js:44]: This should be `debug: true`. Later I will make it pass the options into this function.

File: tasks/bundle-html.js [hunk 7]
```diff
@@ -98,7 +98,7 @@ async function rebuildHTML(changedFiles) {
     await Promise.all(
         pages
             .filter((page) => changedFiles.some((changed) => changed === getSrcPath(page.cwdPath)))
-            .map((page) => bundleHTMLPage(page, {production: false}))
+            .map((page) => bundleHTMLPage(page, {debug: false}))
```
Comment (alexanderby) [inline tasks/bundle-html.js:101]: This should be `debug: true`.

File: tasks/bundle-js.js [hunk 8]
```diff
@@ -100,27 +100,27 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
...
                     compilerOptions: {
-                        removeComments: production ? true : false,
-                        sourceMap: production ? false : true,
+                        removeComments: debug ? false : true,
+                        sourceMap: debug ? true : false,
                     },
                 },
-                clean: production ? true : false,
-                cacheRoot: production ? null : `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache`,
+                clean: debug ? false: true,
```
Comment (alexanderby) [inline tasks/bundle-js.js:107]: Run the `code-style`, it should fix it.

File: tasks/bundle-js.js [hunk 9]
```diff
@@ -100,27 +100,27 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
...
+                cacheRoot: debug ? `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache` : null,
             }),
             rollupPluginReplace({
-                '__DEBUG__': production ? 'false' : 'true',
-                '__PORT__': production ? '-1' : String(PORT),
+                '__DEBUG__': watch ? 'true' : 'false',
```
Comment (alexanderby) [inline tasks/bundle-js.js:111]: Use `debug` boolean value here

File: tasks/bundle-js.js [hunk 10]
```diff
@@ -100,27 +100,27 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
...
+                '__PORT__': watch ? String(PORT) : '-1',
```
Comment (alexanderby) [inline tasks/bundle-js.js:112]: Add a 3rd `__WATCH__` parameter here and use if to enable or disable the WebSocket connection.

File: tasks/bundle-locales.js [hunk 11]
```diff
@@ -51,7 +51,7 @@ module.exports = createTask(
     ['src/_locales/**/*.config'],
     async (changedFiles) => {
         for (const file of changedFiles) {
-            await bundleLocale(file, {production: false});
+            await bundleLocale(file, {debug: false});
```
Comment (alexanderby) [inline tasks/bundle-locales.js:54]: This should be `debug: true`.

File: tasks/copy.js [hunk 12]
```diff
@@ -55,8 +55,8 @@ module.exports = createTask(
     async (changedFiles) => {
         for (const file of changedFiles) {
             if (await fs.exists(file)) {
-                await copyFile(file, {production: false, firefox: false});
-                await copyFile(file, {production: false, firefox: true});
+                await copyFile(file, {debug: false, firefox: false});
```
Comment (alexanderby) [inline tasks/copy.js:58]: This should be `debug: true`.

File: tasks/copy.js [hunk 13]
```diff
@@ -55,8 +55,8 @@ module.exports = createTask(
...
+                await copyFile(file, {debug: false, firefox: true});
```
Comment (alexanderby) [inline tasks/copy.js:59]: This should be `debug: true`.

**Pull request merged**